### PR TITLE
feat(event-filtering): Add LastPass error to browser extension filter

### PR DESCRIPTION
**Pull request by lobsterkatie**
Full error message from an event we'd like to block:

`l.LegacyGlobal.should_do_lastpass_here is not a function. (In 'l.LegacyGlobal.should_do_lastpass_here(document)', 'l.LegacyGlobal.should_do_lastpass_here' is undefined)
`

### CODE REVIEW

**Comment by lobsterkatie** [comment]
`should_do_lastpass_here` seemed like a function name only LP itself would choose*, so I went with that in terms of how specific to be. Open to feedback on that choice. 

*Not that this is the world's _best_ metric, but I searched all of GitHub, and the only instances of `should_do_lastpass_here` I found were three people who managed to not only capture but also commit the entirety of the LP source code (likely by saving a webpage as text while running LP in their browser).

File: src/sentry/message_filters.py [hunk 1]
```diff
@@ -235,6 +235,8 @@ def _localhost_filter(project_config, data):
         # Dragon Web Extension from Nuance Communications
         # See: https://forum.sentry.io/t/error-in-raven-js-plugin-setsuspendstate/481/
         'plugin.setSuspendState is not a function',
+        # lastpass
+        'should_do_lastpass_here'
```
Comment (mattrobenolt) [inline src/sentry/message_filters.py]: Add a trailing comma plz.

**Review by mattrobenolt — APPROVED** [review]

File: src/sentry/message_filters.py (hunk 1 above)
Comment (lobsterkatie) [reply src/sentry/message_filters.py]: Shoot! Meant to. Will do. Thanks for catching that.

Comment (lobsterkatie) [inline src/sentry/message_filters.py]:
```suggestion
        'should_do_lastpass_here',
```

**Pull request merged**